CMakeLists.txt

```
@@ -334,19 +336,19 @@ set_option(SDL_PTHREADS            "Use POSIX threads for multi-threading" ${SDL
 dep_option(SDL_PTHREADS_SEM        "Use pthread semaphores" ON "SDL_PTHREADS" OFF)
 dep_option(SDL_OSS                 "Support the OSS audio API" ${SDL_OSS_DEFAULT} "UNIX_SYS OR RISCOS;SDL_AUDIO" OFF)
 dep_option(SDL_ALSA                "Support the ALSA audio API" ${UNIX_SYS} "SDL_AUDIO" OFF)
-dep_option(SDL_ALSA_SHARED         "Dynamically load ALSA audio support" ON "SDL_ALSA" OFF)
+dep_option(SDL_ALSA_SHARED         "Dynamically load ALSA audio support" ${SDL_DEPS_SHARED} "SDL_ALSA" OFF)
```

How about doing it like this?
Then re-configuring with `-DSDL_DEPS_SHARED=OFF` will force all deps to off.
```suggestion
dep_option(SDL_ALSA_SHARED         "Dynamically load ALSA audio support" ON "SDL_DEPS_SHARED;SDL_ALSA" OFF)
```

---

Sure, can do this, only the outcome matters. :)